src/core_modules/capture-core/components/Pages/common/TEIRelationshipsWidget/RegisterTei/RegisterTei.component.tsx

```diff
@@ -1,29 +1,22 @@
-// @flow
-import React, { type ComponentType, useContext, useCallback } from 'react';
+import React, { useContext, useCallback } from 'react';
 import { compose } from 'redux';
-import { withStyles } from '@material-ui/core/styles';
+import { withStyles, type WithStyles } from '@material-ui/core/styles';
 import i18n from '@dhis2/d2-i18n';
 import { Button } from '@dhis2/ui';
 import { RegisterTeiDataEntry } from './DataEntry/RegisterTeiDataEntry.container';
 import { RegistrationSection } from './RegistrationSection';
 import { DataEntryWidgetOutput } from '../../../../DataEntryWidgetOutput/DataEntryWidgetOutput.container';
-import { ResultsPageSizeContext } from '../../../shared-contexts';
-import type { ComponentProps } from './RegisterTei.types';
+import { ResultsPageSizeContext, type ResultsPageSizeContextType } from '../../../shared-contexts';
 import { withErrorMessageHandler } from '../../../../../HOC';
+import { ComponentProps, getStyles } from './RegisterTei.types';
```

Comment:
Never put styles in a types file. Styles should always live in the same file as the component. However, there are a lot of other types in this file that can be moved over.